tests/integration/ha/continuous_writes.py

```diff
@@ -256,7 +263,7 @@ def _client(_data) -> OpenSearch:
         # write last expected written value on disk
         with open(ContinuousWrites.LAST_WRITTEN_VAL_PATH, "w") as f:
             if is_bulk:
-                write_value = (100 * write_value) + 99
+                write_value += 99
```

Comment:
What we had was correct, we're ensuring that this **only** gets incremented if `write_value` has been successfully incremented.

Reply:
@Mehdi-Bendriss not sure I follow?? Originally, we [used to have write_value starting as 0 (an int)](https://github.com/canonical/opensearch-operator/blob/de29af17d395d50e4c3fa2c97f1cdc4178d4981d/tests/integration/ha/continuous_writes.py#L181C45-L181C46):
```
>>> write_value = 0
>>> write_value = (100 * write_value) + 99
>>> write_value
99
```

This `write_value += 99` does effectively the same thing, BUT it fixes a problem: if we have a backup recovered, `series_index` will start from a non-zero value and w we want to have: `max-id`, `write_value` and `last_expected_id` all matching nonetheless as soon as we restart `cwrites`.

If a failure happens and no docs are written, then `written_value (99 now) != 0`, it will fail.

So, I was having a situation where a recovered `series_index` starts with, say, 101 docs. If we add 10 more docs and run the original formula: `write_value == 11100` now.

Reply:
hmmm i understand, this was indeed not designed to have a "resume" functtion